src/ui/class-block-editor.php

```diff
@@ -132,4 +140,32 @@ public function get_rewrite_republish_permalink() {
 
 		return $this->link_builder->build_rewrite_and_republish_link( $post );
 	}
+
+	/**
+	 * Generates a URL to the original post edit screen.
+	 *
+	 * @return string The URL. Empty if the copy post doesn't have an original.
+	 */
+	public function get_original_post_edit_url() {
+		$post = \get_post();
+
+		if ( \is_null( $post ) ) {
+			return '';
+		}
+
+		$original_post_id = Utils::get_original_post_id( $post->ID );
```

Comment:
See above about `Utils::get_original_post_id` when we already have `Utils::get_original`